package.json

```diff
@@ -47,7 +47,7 @@
   },
   "scripts": {
     "bootstrap:hoist": "lerna bootstrap --hoist --nohoist=one-cerner-style-icons",
-    "clean": "lerna clean --yes && rm -rf node_modules",
+    "clean": "lerna clean --yes && rm -rf node_modules || true",
```

Comment:
this was added to allow this check to succeed even if lerna isn't installed.... no need to clean then right?